resolwe/flow/tests/test_models.py

```diff
@@ -645,6 +675,10 @@ def test_save_storage_file(self):
             contributor=self.contributor,
             process=self.proc,
         )
+        data_location = DataLocation.objects.create(subpath='')
```

Comment:
Is data location required for testing storage?

Reply:
Yes, somewhere along the way `location.get_path` gets called.